modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/behavior/TextFieldBehavior.java

```diff
@@ -178,7 +178,8 @@ public void setTextFieldSkin(TextFieldSkin skin) {
     @Override protected void fire(KeyEvent event) {
         TextField textField = getNode();
         EventHandler<ActionEvent> onAction = textField.getOnAction();
-        ActionEvent actionEvent = new ActionEvent(textField, null);
+        // fix JDK-8207774: use textField as target to prevent immediate copy in dispatch
```

Comment:
Minor: I don't think there is a need to reference the bug ID here (the rest of the comment is good)